tests/nested/classic/hotplug/task.yaml

```diff
@@ -55,12 +55,7 @@ execute: |
     hotplug_add_dev1
 
     # sanity checks to make sure qemu setup is correct
-    for _ in $(seq 5); do
-        if nested_exec "udevadm info -e" | MATCH "ID_MODEL=QEMU_USB_SERIAL"; then
-            break
-        fi
-        sleep 1
-    done
+    nested_retry "--wait 1 -n 5 sh -c 'udevadm info -e | grep -qE \"ID_MODEL=QEMU_USB_SERIAL\"'"
```

Comment:
There should be no need for the outer quotes (`"`); do you get some error if you remove them?

Reply:
I tried that first, but then I need to scape all the ' and " then, so I preferred to send the params like this because it is simpler to read and write

Reply:
No, you don't always need to quote them, but you might need to escape the `|`:

        nested_retry --wait 1 -n 5 sh -c 'udevadm info -e \| grep -qE "ID_MODEL=QEMU_USB_SERIAL"'